src/jats/__tests__/__snapshots__/jats-importer.test.ts.snap

```diff
@@ -4325,8 +4325,8 @@ Array [
   Object {
     "_id": "MPTableElementFooter:test",
     "containedObjectIDs": Array [
-      "MPParagraphElement:test",
       "MPFootnotesElement:test",
+      "MPParagraphElement:test",
```

Comment:
I was expecting an object of type `GeneralTableFootnote` instead.

Reply:
Is there a need for a new model? I think we can have the encoder wrap them appropriately.